net/sing-box/Makefile

```diff
@@ -66,9 +66,6 @@ define Package/sing-box/config
 			bool "Build with gVisor support"
 			default y
 
-		config SINGBOX_WITH_LWIP
-			bool "Build with LWIP Tun stack support"
-
```

Comment:
Its great though that you copy pasted changelog from their tag release, but in your commit message there isnt anything why this is removed and you can not expect anyone that from your snipper he goes to link to read it more. 

So, it is completely fine to trim down the commit description.

1. LWIP was removed due to XY

Release notes: LINK

Thats it. If someone wants to read about several news or bug fixes, he can open the web browser to check it out.

Reply:
Missing descriptions have been added 😺